using_images/s2i_images/java.adoc

```diff
@@ -13,49 +13,50 @@ toc::[]
-resulting image can be run on {product-title} or run directly with Docker.
-
-The builder image is intended for use with
+{product-title} provides
+xref:../../architecture/core_concepts/builds_and_image_streams.adoc#source-build[S2I builder images] for building Java applications. These builder images take your application source or binary artifacts, build the source using Maven (if the source was provided), and assemble the artifacts with any required dependencies to create a new, ready-to-run image containing your Java application. This resulting image can be run on {product-title} or run directly with Docker.
+
+The builder images are intended for use with
 link:https://maven.apache.org[Maven]-based Java standalone projects that are run
 via main class.
 
 
 [[s2i-images-java-versions]]
 == Versions
 
-The current version of the Java S2I builder image supports OpenJDK 1.8, Jolokia
-1.3.5, and Maven 3.3.9-2.8.
+The current version of the Java S2I builder images support OpenJDK 1.8 and 11,
+Jolokia 1.6.2, and Maven 3.6.
 
 
 [[s2i-images-java-images]]
 == Images
 
-The RHEL 7 image is available through the Red Hat Registry:
+The RHEL 7 and RHEL 8 images are available through the Red Hat Registry:
+
+*RHEL 7 based images*
 
 ----
 $ docker pull registry.redhat.io/redhat-openjdk-18/openjdk18-openshift
+$ docker pull registry.redhat.io/openjdk/openjdk-11-rhel7
+----
+
+*RHEL 8 based images*
+
+----
+$ docker pull registry.redhat.io/ubi8/openjdk-8
+$ docker pull registry.redhat.io/ubi8/openjdk-11
 ----
 
 ifdef::openshift-online[]
 You can use the image through the `redhat-openjdk18-openshift` image stream.
 endif::openshift-online[]
 
 ifndef::openshift-online[]
-To use this image on {product-title}, you can either access it directly from
-the Red Hat Registry or push it into your
-xref:../../install_config/registry/index.adoc#install-config-registry-overview[{product-title}
-container image registry].  Additionally, you can create an
-xref:../../architecture/core_concepts/builds_and_image_streams.adoc#image-streams[image
-stream] that points to the image, either in your container image registry or at the
-external location. Your {product-title} resources can then reference the
-link:https://github.com/jboss-openshift/application-templates/blob/master/jboss-image-streams.json[image
-stream definition].
+To use these images on {product-title}, you can either access them directly from
+the Red Hat Registry or push them into your
+xref:../../install_config/registry/index.adoc#install-config-registry-overview[{product-title} container image registry].  Additionally, you can create an
+xref:../../architecture/core_concepts/builds_and_image_streams.adoc#image-streams[image stream] that points to the image, either in your container image registry or at the external location. Your {product-title} resources can then reference the
+link:https://github.com/jboss-openshift/application-templates/blob/master/jboss-image-streams.json[image stream definition].
```

Comment:
@jmtd This link is broken. Do you know the correct link?

Reply:
Good catch. This should work: https://raw.githubusercontent.com/jboss-container-images/openjdk/release/templates/image-streams.json

Reply:
Thanks.